dist_list.txt

```diff
@@ -1 +1 @@
-el6 el7 fc20 fc21
\ No newline at end of file
+el6 el7 fc20 fc21 fc22
```

Comment:
I think we should also drop support for F20, since it is only supported for two more days.